sdk/lib/3rdparty/freetype/CMakeLists.txt

```diff
@@ -43,14 +37,15 @@ list(APPEND SOURCE
     src/pfr/pfr.c
     src/psaux/psaux.c
     src/pshinter/pshinter.c
-    src/psnames/psmodule.c
+    src/psnames/psnames.c
     src/raster/raster.c
     src/sfnt/sfnt.c
     src/smooth/smooth.c
     src/truetype/truetype.c
     src/type1/type1.c
     src/type42/type42.c
-    src/winfonts/winfnt.c)
+    src/winfonts/winfnt.c
+    freetype.rc)
```

Comment:
This .rc file can be removed, because the freetype library compiled is a static library, that is linked into the ftfd.dll (see win32ss/drivers/font/ftfd ) and that latter one already has its own freetype.rc file.

Reply:
Deleted `freetype.rc` in `freetype`.